src/java.base/share/classes/java/util/JapaneseImperialCalendar.java

```diff
@@ -1116,11 +1116,9 @@ public int getMinimum(int field) {
      */
     public int getMaximum(int field) {
         switch (field) {
-        case YEAR:
-            {
+            case YEAR -> {
                 // The value should depend on the time zone of this calendar.
-                LocalGregorianCalendar.Date d = jcal.getCalendarDate(Long.MAX_VALUE,
-                                                                     getZone());
+                LocalGregorianCalendar.Date d = jcal.getCalendarDate(Long.MAX_VALUE, getZone());
                 return Math.max(LEAST_MAX_VALUES[YEAR], d.getYear());
             }
         }
```

Comment:
A switch with one element here is kind of weird.  I would turn this into "return switch (field) { ... }", with two cases, YEAR and default.

Reply:
Done, also at line 1169